resources/views/scheduleInstrument.html

```diff
@@ -0,0 +1,427 @@
+<div id="topContainer">
+    <div id="projectDropDown"></div>
+    <br/>
+    <div id="instrumentDropDown"></div>
+    <br/>
+    <div id="instrumentOperatorDropDown"></div>
+    <br/>
+    <div id="paymentMethodDropDown"></div>
+    <a href="javascript:void(0);" id="addPaymentMethod">[Add Payment Method]</a>
+    <br/>
+
+</div>
+
+<div id="calendarWrapper" style="min-height: 500px;">
+
+    <div id="calendar">
+        Loading...
+    </div>
+
+    <div class="modal fade" id="event-modal">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title">Add Instrument Time</h5>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                        <span aria-hidden="true">&times;</span>
```

Comment:
Misplaced `&`?

Reply:
no, this is the 'X' sign from html entities in the window modal.